htdocs/API/2.0/index.php

```diff
@@ -8,14 +8,12 @@
 use CsrDelft\controller\api\ApiMaaltijdenController;
 use Jacwright\RestServer\RestServer;
 
+require_once 'configuratie.include.php';
+
 /**
- * Maak de API toegankelijk vanaf bepaalde externe Origins.
+ * Maak de API toegankelijk vanaf bepaalde externe domeinen.
```

Comment:
Dit "toegankelijk" is alleen client side (dit is server side code) en de term "bruikbaar" is geschikter omdat je alsnog een inhoudelijk antwoord van de server krijgt, alleen zal de browser van de client de content tegenhouden.